playbooks/cloud-provider-openstack-e2e-test-csi-cinder/run.yaml

```diff
@@ -91,14 +91,21 @@
               INSTANCE_UUID=$(curl http://169.254.169.254/openstack/latest/meta_data.json | python -c "import sys, json; print json.load(sys.stdin)['uuid']")
           fi
           echo "$INSTANCE_UUID" > /var/lib/cloud/data/instance-id
-          export VERSION=latest
-          # Build latest images from source
-          make image-cinder-csi-plugin
 
-          if [[ -z "$(docker images -q k8scloudprovider/cinder-csi-plugin:$VERSION 2> /dev/null)" ]]; then
-              # add tag without architecture so we don't need to push manifests
-              docker image tag k8scloudprovider/cinder-csi-plugin-$ARCH:$VERSION k8scloudprovider/cinder-csi-plugin:$VERSION
-          fi
+          # Build latest images from source
+          cat << EOF > Dockerfile
+          # Based on centos
+          FROM centos:7.4.1708
```

Comment:
@lingxiankong I dont think this would be recommended method , centos doesnt work for some tests due to missing utilities and we have shifted to debian. I prefer we use the same dockerfile as in repo

Reply:
@ramineni make sense, I will change this when I am available

Reply:
@ramineni Please review https://github.com/kubernetes/cloud-provider-openstack/pull/1045, after that, we could keep using `make image-csi-plugin` for both previous branches and master.